requirements_test.txt

```diff
@@ -1,8 +1,9 @@
-# requirements for running tests
+# Requirements for running tests and non-production Docker build.
 # Expects some apt packages are already installed.
 # see .github/workflows/python_tests.yml for more details.
 
 -r requirements.txt
+debugpy>=1.2.0
 flake8==3.9.0
 mypy==0.812
 pymemcache==3.4.1
```

Comment:
Is `pymemcache` _only_ used for testing?

Reply:
I think sorta; during testing, if the memcache servers aren't running it "falls back" to pymemcache as an alternative.